pure.zsh

```diff
@@ -702,6 +702,9 @@ prompt_pure_state_setup() {
 	# Show `username@host` if inside a container and not in GitHub Codespaces.
 	[[ -z "${CODESPACES}" ]] && prompt_pure_is_inside_container && username='%F{$prompt_pure_colors[user]}%n%f'"$hostname"
 
+        # Show `⬢` if inside a toolbox.
+        [[ "$HOSTNAME" = "toolbox" ]] && [[ $(which toolbox) = "/usr/bin/toolbox" ]] && username='%F{$prompt_pure_colors[host]}⬢%f'
```

Comment:
We use tabs for indentation.

Would it not also make sense to have the `prompt_pure_is_inside_container` check pass since Toolbox is container based?

Reply:
I think Toolbox wouldn't pass that tes, since it's not based on Docker/LXC.

![example](https://user-images.githubusercontent.com/47647695/154533526-9c6922f4-5f80-4df9-974f-6d392edb0ce7.png)

Reply:
Putting this check makes the diamond not render anymore. I suspect it is caused by the patchy way Toolbox works.
![example](https://user-images.githubusercontent.com/47647695/154543164-e637959a-e73b-4345-9ccd-43f3c10d4457.png)

Reply:
Inserting that check makes it not work anymore. I suspect it is caused by the patchy way Toolbox works.
![example](https://user-images.githubusercontent.com/47647695/154543520-d638f8a8-ae37-4dc8-95dd-af077a70c859.png)